v2/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/zmap/zlint/v2
 
 require (
+	github.com/olekukonko/tablewriter v0.0.4
```

Comment:
I'm a little bit hesitant to see a third party dependency added for something that's limited in functionality to the `zlint` binary. Do you think it would be difficult to implement the table output using `fmt` and the stdlib? I might be overlooking the reason you choose this dep.

Reply:
I agree.  I picked this because it was easy, although in the end the output was simplistic enough that is does seem like overkill and not worth the risk of relying on a third-party dependancy. 

I'll make a new PR without this dependancy.